features/steps/jobs_steps.go

```diff
@@ -71,6 +79,8 @@ func NewJobsFeature(mongoFeature *componentTest.MongoFeature) (*JobsFeature, err
 
 	f.MongoClient = mongodb
 
+	f.AuthHandler = &mock.AuthHandlerMock{}
```

Comment:
So we shouldn't be mocking the auth client but instead mock responses from the service that the auth client will communicate with, I believe this is Zebedee - so we need to build a fake Zebedee service that the auth client will talk to.

Reply:
Reasoning for this is that these tests will pass, even if the client we are using in this app is broken/wrong.

If we are mocking responses from Zebedee in these tests and then we switch out the client for another one these tests will help confirm the app is working as expected based on our understanding of the Zebedee API interface.

Reply:
The auth client is not part of this app so, although I understand the reasoning, I don't think I can mock something that is not being used directly by this service?